src/sentry/remote_config/docs/api.md

```diff
@@ -117,3 +117,41 @@ Set the DSN's configuration.
 Delete the DSN's configuration.
 
 - Response 204
+
+## Configuration Proxy [/relays/<project_id>/configuration/]
```

Comment:
As outlined in Slack, this route needs to match the route which is exposed from Relay.

Reply:
Can you expand on this?  What should the route look like?  What's being exposed in Relay?

Reply:
You exposed the route `/api/:project_id/configuration/` from Relay, Sentry should implement the same interface as Relay. You're adding it here as a Relay specific route.

Reply:
I thought we didn't need to expose a route in Relay and it would forward automatically?

Reply:
I don't know how else to put it, this is not a Relay specific API, so it should not be in the Relay specific API routes.

Reply:
I understand now.  Its a categorization problem not a functionality problem.  Yes, I will re-scope the route to match a more normal pattern.